hoomd/SnapshotSystemData.cc

```diff
@@ -39,6 +39,40 @@ void SnapshotSystemData<Real>::replicate(unsigned int nx, unsigned int ny, unsig
     pair_data.replicate(n, old_n);
     }
 
+template<class Real> void SnapshotSystemData<Real>::wrap()
+    {
+    vec3<double> frac;
+    vec3<double> wrapped;
+    vec3<double> pos;
+    vec3<double> fin;
+    int3 img;
+
+    for (unsigned int i = 0; i < particle_data.size; i++)
+        {
+        pos = particle_data.pos[i];
+        frac = global_box.makeFraction(particle_data.pos[i]);
+        wrapped = vec3<double>(
+            fmod(fmod(frac.x, 1.0) + 1.0, 1.0),
+            fmod(fmod(frac.y, 1.0) + 1.0, 1.0),
+            fmod(fmod(frac.z, 1.0) + 1.0, 1.0)
+            );
```

Comment:
1. Use a lambda to define the math expression only once.
2. Be explicit with the namespace `std::`. Someplace in this code has `using namespace std` but this is [considered bad practice](https://www.geeksforgeeks.org/using-namespace-std-considered-bad-practice/).
3. Use the template parameter `Real` precision unless @joaander disagrees.
```suggestion
        auto modulus_positive = [](auto x){ return std::fmod(std::fmod(x, Real(1.0)) + Real(1.0), Real(1.0)); };
        wrapped = vec3<Real>(
            modulus_positive(frac.x),
            modulus_positive(frac.y),
            modulus_positive(frac.z)
            );
```